2200005/10505.py

```diff
@@ -0,0 +1,9 @@
+a= int(input())
+for i in range(a):
+    num = int(input())
+    cnt = 0
+    l = list(map(int,input().split()))
+    for j in l:
+        if j<=sum(l)/num:
+            cnt+=1
```

Comment:
깔끔하네요! 👍
```suggestion
    avg = sum(l) / num
    for j in l:
        if j <= avg:
            cnt += 1
```
평균 값을 비교할 때마다 매 번 계산하지 않고, 한 번만 계산하는 방법도 있습니다 😄